working/bazel/module-structure.md

```diff
@@ -0,0 +1,234 @@
+The following is a reference for [`Bazel`](https://bazel.build/) (internally
```

Comment:
internally -> internally in Google

(This repository is not aimed at Googlers only).
Consider whether it's important to external readers at all, it probably isn't.

Much of the document seems to be about Google-only design choices, so maybe instead make it explicit that this document is mainly about *Blaze at Google*, but that it also applies to Bazel build systems.

Reply:
Done.

(Bazel is currently Google only for the most part, but that is _rapidly_ changing. For example SpaceX uses it as their own build system, and many other medium/larger companies are considering it)